bridge/setu/listener/rootchain.go

```diff
@@ -98,28 +98,20 @@ func (rl *RootChainListener) ProcessHeader(newHeader *blockHeader) {
 
 	requiredConfirmations := rootchainContext.ChainmanagerParams.MainchainTxConfirmations
 	headerNumber := newHeader.header.Number
-	from := headerNumber
 
-	// If incoming header is a `finalized` header, it can directly be considered as
-	// the upper cap (i.e. the `to` value)
-	//
-	// If incoming header is a `latest` header, rely on `requiredConfirmations` to get
-	// finalized block range.
-	if !newHeader.isFinalized {
-		// This check is only useful when the L1 blocks received are < requiredConfirmations
-		// just for the below headerNumber -= requiredConfirmations math operation
-		confirmationBlocks := big.NewInt(0).SetUint64(requiredConfirmations)
-		if headerNumber.Cmp(confirmationBlocks) <= 0 {
-			rl.Logger.Error("Block number less than Confirmations required", "blockNumber", headerNumber.Uint64, "confirmationsRequired", confirmationBlocks.Uint64)
-			return
-		}
+	// This check is only useful when the L1 blocks received are < requiredConfirmations
+	// just for the below headerNumber -= requiredConfirmations math operation
+	confirmationBlocks := big.NewInt(0).SetUint64(requiredConfirmations)
+	if headerNumber.Cmp(confirmationBlocks) <= 0 {
+		rl.Logger.Error("Block number less than Confirmations required", "blockNumber", headerNumber.Uint64, "confirmationsRequired", confirmationBlocks.Uint64)
```

Comment:
Just for more clarity, can we change this to
```
rl.Logger.Error("Ethereum block number minor than required number of confirmations", "blockNumber", headerNumber.Uint64, "requiredConfirmations", confirmationBlocks.Uint64)
```

Also, do you see this as an error or can we downgrade to `Warn` ?

Thanks

Reply:
I think we can ignore this case for now as this will never occur on mumbai and mainnet, It will only happen when the supporting L1 has block number less than 64, That is only in devnets I think for now.

Still if you want I can change the message and downgrade to warn.